### PR TITLE
ENG-3379 :  Replace coverall code coverage with 'CodeCov' in Sightmachine-SDK

### PR DESCRIPTION
**Pull request by JMkrish**
https://sightmachine.atlassian.net/browse/ENG-3379

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -58,10 +59,15 @@ jobs:
       - run:
           name: Run Unit Tests
           command: |
+            set -x
+            pip install -r requirements-coveralls.txt
```
Comment (mklein0) [inline .circleci/config.yml:63]: May want to rename the requirements file as the application used is no longer coveralls.  requirements-codecoverage.txt?

File: smsdk/ma_session.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def _get_records(
         self,
         endpoint: str,
         method: str = "get",
-        _limit: float = np.Inf,
+        _limit: int = np.Inf,
```
Comment (mklein0) [inline smsdk/ma_session.py:47]: Can not set an integer to Infinite.  Can only set it to `None` for an invalid number.

File: .circleci/config.yml (hunk 1 above)
Comment (JMkrish) [reply .circleci/config.yml]: Renamed 'requirements-coveralls.txt' as 'requirements-codecoverage.txt'

**Review by mklein0 — APPROVED** [review]
Please place a link/bookmark to the Ticket in the Description of the PR.  It makes navigating between applications easier.  Otherwise looks good.

**Pull request merged**